version.php

```diff
@@ -26,8 +26,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2025022000;
-$plugin->release = 2019080801; // Match release exactly to version.
-$plugin->requires = 2016052318; // 3.1.18+.
```

Comment:
Seeing as we're not actually doing anything that would affect functionality, and also there are no database schema changes or anything that would require a purge cache, I'd be keeping the versions as-is and not changing them

Reply:
I will change it back